src/azure-cli/azure/cli/command_modules/sqlvm/custom.py

```diff
@@ -290,7 +290,7 @@ def sqlvm_create(client, cmd, sql_virtual_machine_name, resource_group_name, sql
 def sqlvm_update(cmd, instance, sql_virtual_machine_name, resource_group_name, sql_server_license_type=None,
                  sql_image_sku=None, least_privilege_mode=None, enable_auto_patching=None,
                  day_of_week=None, maintenance_window_starting_hour=None, maintenance_window_duration=None,
-                 enable_auto_backup=None, enable_encryption=False, retention_period=None, storage_account_url=None,
+                 enable_auto_backup=None, enable_encryption=False, retention_period=None, storage_account_url=None, prompt=True,
```

Comment:
Please add `pylint: disable=unused-argument` to pass CI style check

Reply:
done